src/services/section-lieu.js

```diff
@@ -0,0 +1,61 @@
+import $ from 'jquery';
+
+export const sectionLieu = () => {
```

Comment:
Need to add arguments (arr, index), because we will need it for the random buttons et with the json server